cmake/scripts/common/ArchSetup.cmake

```diff
@@ -11,6 +11,9 @@
 # DEP_DEFINES - compiler definitions for system dependencies (e.g. LIRC)
 # + the results of compiler tests etc.
 
+# workaround a bug in older cmake, where binutils wouldn't be set after deleting CMakeCache.txt
+include(CMakeFindBinUtils)
```

Comment:
may be add a check for cmake version?

Reply:
I'd rather not, as doing this doesn't hurt. If really needed, we can make it dependent on the variables being empty.